UI/Tw.html

```diff
@@ -0,0 +1,496 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+      <meta name="keywords" content="HTML, BSU training, website">
+    <title>Thinhill</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+          <span>
+              <p> Thinhill</p>
+          </span>
+          <div class="hSp1">
+             Hello <br> Nadia!
+           </div>
+          <span>
+              <p><b><u><a href="out.html" target="_blank">Sign out</a></u></b></p>
+          </span>
+      </header>
+	  
+	  
+      <main>
+        <aside class="column right-aside">
```

Comment:
I'd rather use 'section' tag here. And also it doesn't seem to be on the right side 😉